input/ipc-unix.c

```diff
@@ -105,7 +105,9 @@ static void *client_thread(void *p)
     struct client_arg *arg = p;
     bstr client_msg = { talloc_strdup(NULL, ""), 0 };
 
-    mpthread_set_name(arg->client_name);
+    char *tname = talloc_asprintf(NULL, "ipc/client/%s", arg->client_name);
```

Comment:
`pthread_setname_np` has a limit of 16 bytes (including the nul-byte). This is getting too close to that limit.

Reply:
changes to `ipc/%s`, I don't want to have unprefixed random strings in thread name, so at lest `ipc/` should be there.